config/testdata/conf.good.yml

```diff
@@ -109,7 +109,8 @@ scrape_configs:
 - job_name: service-kubernetes
 
   kubernetes_sd_configs:
-  - api_servers:
+  - kind: service_endpoints
```

Comment:
It's more that this may be used as an example by users to get things going, and endpoints is not something we should be implicitly pointing users towards.